implementation/src/main/java/io/smallrye/openapi/runtime/io/OpenApiParser.java

```diff
@@ -483,7 +502,7 @@ private Components readComponents(JsonNode node) {
      * 
      * @param node
      */
-    private Schema readSchema(JsonNode node) {
+    private SchemaImpl readSchema(JsonNode node) {
```

Comment:
Do you have to use the impl ? No way we can just use the interface ?

Reply:
OK because you have to set the name ?

Reply:
I did that in the past for handy access to `getName`.